### PR TITLE
Add unit test for OpenIdContextBean

### PR DESCRIPTION
**Pull request by utle**
Added unit tests for OpenIdContextBean

### CODE REVIEW

**Comment by utle** [comment]
#build

File: dev/io.openliberty.security.jakartasec.3.0.internal.cdi/test/io/openliberty/security/jakartasec/cdi/extensions/OpenIdContextBeanTest.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        Set<Type> types = openIdContextBean.getTypes();
+        String expectedTypeString = new TypeLiteral<OpenIdContext>() {
+        }.getType().toString();
+
+        assertEquals("Must be a TypeLiteral<OpenIdContextBean> type.", expectedTypeString, types.iterator().next().toString());
+    }
+
+    @Test
+    public void testGetBeanClass() {
+        assertEquals("Must OpenIdContextBean.", OpenIdContextBean.class, openIdContextBean.getBeanClass());
```
Comment (Zech-Hein) [inline dev/io.openliberty.security.jakartasec.3.0.internal.cdi/test/io/openliberty/security/jakartasec/cdi/extensions/OpenIdContextBeanTest.java]:
```suggestion
        assertEquals("Must be OpenIdContextBean.", OpenIdContextBean.class, openIdContextBean.getBeanClass());
```

**Review by Zech-Hein — CHANGES_REQUESTED** [review]
Just adding the "be", everything else looks good to me

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#run-libby-bot

**Review by Zech-Hein — APPROVED** [review]

**Pull request merged**